src/test/java/org/folio/rtac/rest/exceptions/HttpExceptionTest.java

```diff
@@ -8,45 +8,60 @@
 public class HttpExceptionTest {
   @Test
   public final void testHttpExceptionInt() {
-    final HttpException thrown = assertThrows(HttpException.class, () -> {
-      throw new HttpException(5);
-    });
+    final HttpException thrown =
```

Comment:
Looks like the only differences in this file are formatting. Please consider reformatting changes for a separate PR. If there are valid changes, they are difficult to spot amidst formatting changes.

Reply:
Done.